site/examples/embeds.js

```diff
@@ -83,20 +65,18 @@ const VideoElement = ({ attributes, children, element }) => {
             }}
           />
         </div>
-        {selected && focused ? (
-          <input
-            value={url}
-            onClick={e => e.stopPropagation()}
-            style={{
-              marginTop: '5px',
-              boxSizing: 'border-box',
-            }}
-            onChange={value => {
-              const path = ReactEditor.findPath(editor, element)
-              Transforms.setNodes(editor, { url: value }, { at: path })
-            }}
-          />
-        ) : null}
+        <input
+          value={url}
+          onClick={e => e.stopPropagation()}
+          style={{
+            marginTop: '5px',
+            boxSizing: 'border-box',
+          }}
+          onChange={e => {
+            const path = ReactEditor.findPath(editor, element)
+            Transforms.setNodes(editor, { url: e.target.value }, { at: path })
+          }}
```

Comment:
the event handler was quite busted because it was expecting a value string but receiving an input event, and Transforms wasn't being imported.